.github/workflows/ci.yml

```diff
@@ -8,114 +8,72 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v2
-      - uses: actions/cache@v2
-        with:
-          path: "**/node_modules"
-          key: ${{ runner.os }}-${{ hashFiles('yarn.lock') }}-${{ hashFiles('**/package.json') }}
-          restore-keys: |
-            ${{ runner.os }}-${{ hashFiles('yarn.lock') }}-
-            ${{ runner.os }}-
       - uses: actions/setup-node@v2
         with:
-          node-version: "16.13.x"
-      - name: Install package dependencies
+          cache: yarn
+          node-version-file: .nvmrc
+          registry-url: https://npm.pkg.github.com/
+      - run: yarn install --frozen-lockfile
         env:
-          PACKAGE_REGISTRY_ACCESS_TOKEN: ${{ secrets.PACKAGE_REGISTRY_ACCESS_TOKEN }}
-        run: |
-          npm config set "//npm.pkg.github.com/:_authToken" "$PACKAGE_REGISTRY_ACCESS_TOKEN"
-          yarn install --frozen-lockfile
+          NODE_AUTH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
These install steps had previously been attempting to use the `PACKAGE_REGISTRY_ACCESS_TOKEN` org secret to fetch internal packages from the GitHub package registry. However, this secret is (currently) not available to public repos.

This change sets the GitHub actions workflow's own GitHub token as the registry auth token, which will enable the step to install any public packages on the GitHub registry.

Note that internal packages will still not be available — we can cross this bridge when (if?) we come to it.